client/my-sites/earn/ads/wrapper.jsx

```diff
@@ -190,16 +190,23 @@ class AdsWrapper extends Component {
 	}
 
 	renderUpsell() {
-		const { translate } = this.props;
+		const { siteSlug, translate } = this.props;
+		const bannerURL = `/checkout/${ siteSlug }/premium`;
 		return (
-			<UpgradeNudgeExpanded
+			<UpsellNudge
+				callToAction={ translate( 'Upgrade' ) }
 				plan={ PLAN_PREMIUM }
 				title={ translate( 'Upgrade to the Premium plan and start earning' ) }
-				subtitle={ translate(
+				description={ translate(
 					"By upgrading to the Premium plan, you'll be able to monetize your site through the WordAds program."
 				) }
-				highlightedFeature={ FEATURE_WORDADS_INSTANT }
-				benefits={ [
+				feature={ FEATURE_WORDADS_INSTANT }
+				href={ bannerURL }
+				showIcon
+				event="calypso_upgrade_nudge_impression"
+				tracksImpressionName="calypso_upgrade_nudge_impression"
```

Comment:
If we want to keep maintaining the current analytics, I think we also need to set `tracksImpressionProperties`:

https://github.com/Automattic/wp-calypso/blob/7e5c83c7eb3c9da2a76f23e5745231e8fbb43ef7/client/blocks/upgrade-nudge-expanded/index.jsx#L39-L43

We can likely drop the `cta_size` prop though, since this upsell is no longer expanded.

Reply:
Hmm, I'm seeing the correct `cta_feature` prop set with this PR on both impression and click:

<img width="659" alt="Screen Shot 2020-05-05 at 10 09 30 AM" src="https://user-images.githubusercontent.com/2124984/81075588-8da8d480-8eb8-11ea-9de9-8fa8bf54b117.png">

I did update to set `cta_name` to empty since that had changed from the previous version, and set the size to "regular".

Reply:
It's slightly different, we don't need the cta_size prop of "expanded" but let's drop "cta_name". There's slightly different behavior between `undefined` and an empty string

| Before | After |
| ------------- | ------------- |
| <img width="1176" alt="Screen Shot 2020-05-05 at 2 54 57 PM" src="https://user-images.githubusercontent.com/1270189/81120041-c197f000-8ee0-11ea-8503-f718958263a9.png"> | <img width="1191" alt="Screen Shot 2020-05-05 at 2 55 09 PM" src="https://user-images.githubusercontent.com/1270189/81120020-bcd33c00-8ee0-11ea-9394-e8171f5efd4c.png"> |

Reply:
OK, this is the current state:

|Before | After |
| ----- | ----- |
| <img width="608" alt="before" src="https://user-images.githubusercontent.com/2124984/81190459-ded2ca00-8f85-11ea-8d0f-17f33e09bda6.png"> | <img width="595" alt="after" src="https://user-images.githubusercontent.com/2124984/81190474-e4301480-8f85-11ea-9980-70d81f8ad228.png"> |